Common/Tests/Utilities.UI/UI/VisualStudioInstance.cs

```diff
@@ -53,6 +53,22 @@ public class VisualStudioInstance : IDisposable, IVisualStudioInstance {
             _solutionExplorer = _app.OpenSolutionExplorer();
             SelectSolutionNode();
         }
+        
+       //SelectSolutionNode for item visibility 
+        public VisualStudioInstance(SolutionFile solution, VisualStudioApp app, string item)
```

Comment:
item not being used?

Reply:
yes, this is an overload of the ToVs method, and this parameter is used to call the overloaded method.